paper.tex

```diff
@@ -527,7 +525,15 @@ \subsection{Pipeline Design} \label{subsec:pipeline-design}
 
 This procedure is sufficient to generate weakly correct stroke outlines. If the outline contains a cusp -- which can be determined by evaluating the curvature variation along the Euler spiral segment -- we output an evolute patch (as shown in Figure~\ref{fig:evolutes}) in order to achieve strong correctness. A selection of renderings produced by our algorithm that demonstrate both strong and weak correctness are shown in Figure~\ref{fig:stroke_stills}.
 
-A property of the data-parallel structure is that the output is unordered. The cubic-Bézier-to-Euler-spiral conversion is sequential within a single thread, so it is possible to preserve ordering at the scope of an individual path segment, if desired. However, ordering cannot be guaranteed across path segments. We refer to this output data structure as \emph{line soup}. We integrated the compute shader as a stage in a larger compute shader pipeline (discussed in Section \ref{subsection:gpu-results}) that spatially sorts the line soup in subsequent binning and tiling stages prior to rasterization.
+A property of the data-parallel structure is that the output of the compute stage is unordered. The cubic-Bézier-to-Euler-spiral conversion is sequential within a single thread, so it is possible to preserve ordering at the scope of an individual path segment, if desired. However, ordering cannot be guaranteed across path segments. We refer to this output data structure as \emph{line soup}. An application may want to sort the line soup, depending on the requirements of the chosen rendering algorithm. For instance, our own renderer employs tile-based scanline rasterization in a compute shader, with a pipeline architecture that resembles \emph{cudaraster} (\citet{Laine2011}) and \emph{MPVG} (\citet{Ganacim2014}). In this design, the line soup elements are spatially sorted by subsequent compute stages into a grid of 16x16 pixel tiles. The final compute stage of our renderer (called \emph{fine rasterizer}) operates at tile granularity and computes pixel coverage for filled polygons outlined by the line soup (the details of which are beyond the scope of this paper).
```

Comment:
I think we can use citenum here, but it's a style preference.

The ordering feels a little off to me, I'd prefer the following order:
* Shader writes line soup. Line soup is endpoints in viewport coordinate
* Line soup is unordered. Renderer detail as in above paragraph
* In addition to the line variant, we implemented the arc variant